cmd/resourcegen/embed/resources/$cloud_provider$/$filename$.go.tmpl

```diff
@@ -0,0 +1,145 @@
+package {{ .CloudProvider }}
+
+import (
+	"fmt"
+
+	"github.com/shopspring/decimal"
+
+	"github.com/infracost/infracost/internal/resources"
+	"github.com/infracost/infracost/internal/schema"
+)
+
+// TODO: fill in the docs {{ .ResourceName }} with a short description of the cloud resource and
+// add any links to the pricing pages or docs that might be useful to developers in the future.
+// e.g.
+// 		more resource information here: https://aws.amazon.com/{{ .Filename }}/
+//
+//		pricing information here: https://aws.amazon.com/{{ .Filename }}/pricing/
+type {{ .ResourceName }} struct {
+	{{- if .WithHelp }}
+	// Add {{ .ResourceName }} struct properties below that correspond to your resource
+	// and are directly used in cost calculation
+	//
+	// These properties are configured directly from attributes parsed in the IAC provider see
+	// your resource file at: internal/providers/terraform/{{ .CloudProvider }}/{{ .Filename }}.go
+	// for an example in how this is achieved.
+	//
+	// below are a few examples of common properties that are often part of resources:
+	{{- end }}
+	// Address is the unique name of the resource in the IAC language
+	Address       string
+	// Region is the aws region the {{ .ResourceName }} is provisioned within
```

Comment:
Guess we can also replace the provider here :)
```suggestion
	// Region is the {{ .CloudProvider }} region the {{ .ResourceName }} is provisioned within
```

Reply:
fixed in https://github.com/infracost/infracost/pull/1091/commits/a2557a072da0c16e50c972570059b4959b79ff38